common/lanelet2_extension/CMakeLists.txt

```diff
@@ -33,7 +33,8 @@ find_path(PUGIXML_INCLUDE_DIRS
   PATH_SUFFIXES pugixml
 )
 
-find_package(PROJ4 REQUIRED)
+set(PROJ_INCLUDE_DIRS "/usr/include/proj")
```

Comment:
it looks like we are missing libproj-dev in carma-base no?
I wonder where else are we installing it. 
Also if we include the cmake https://github.com/usdot-fhwa-stol/carma-base/blob/develop/noetic-foxy/Dockerfile#L208C69-L208C155

would we be able to keep this as same as previous?

Reply:
I added a similar step in a PR for carma-base. Ubuntu 22.04 uses PROJ instead of PROJ4, so some of these changes are still needed but I removed manually setting the include directories and libraries.
Reference on the PROJ4 to PROJ change: https://proj.org/en/9.5/development/cmake.html